### PR TITLE
32/app title

### LINKED ISSUE #29: Hide Header by default for students

### LINKED ISSUE #30: Provide needed parameters for GET action

### LINKED ISSUE #31: Set default height

### LINKED ISSUE #32: Rename app title in index.html

### LINKED ISSUE #33: When header is hidden, show Wavelength Distribution at the top

### PR DESCRIPTION
**Pull request by pyphilia**
close #29
close #30
close #31
close #32
close #33

### CODE REVIEW

File: src/actions/action.js [hunk 1]
```diff
@@ -45,7 +45,10 @@ const getActions = async (
     }
 
     // create url from params
-    const url = `//${apiHost + ACTIONS_ENDPOINT}?${Qs.stringify(params)}`;
+    const url = `//${apiHost + ACTIONS_ENDPOINT}?${Qs.stringify({
```
Comment (juancarlosfarah) [inline src/actions/action.js:48]: I think this is wrong. We should probably be pushing the current space into the spaceId array of params. Does that make sense?

**Review by juancarlosfarah — CHANGES_REQUESTED** [review]
Looks good. Just a quick question about the params.

**Comment by pyphilia** [comment]
@juancarlosfarah That's actually what was done before, but it seems that since it was a default value, it was not provided back. I changed the process to actually apply `spaceId` to `params` in my new commit (it should be cleaner but let me know).

**Review by hagoptaminian — APPROVED** [review]
Looks good to me. You might want to rename the two occurrences of _Graasp App_ in `manifest.json` as well.

**Review by juancarlosfarah — APPROVED** [review]

**Pull request merged**